app/views/kaminari/_gap.html.haml

```diff
@@ -0,0 +1,2 @@
+%li.page-item.disabled
+  = link_to raw(t 'views.pagination.truncate'), '#', class: 'page-link'
```

Comment:
'#', class: 'page-link'?  it's a comment

Reply:
it is not comment, it is attribute